brianlong/www.validators.app
# search by vote account

#### What's this PR do?
search of validators will also include vote_account

#### How should this be manually tested?
get any validators vote account address and try to search by it. It should still be possible to search by
other params.

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/178277497

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
